tests/fixtures/modifiers.js

```diff
@@ -89,6 +122,28 @@ const modifiersFixtures = [
 		'expected': '(?:(?:ab))',
 		'expectedFlags': '',
 	},
+	{
+		'pattern': '(?i:є)',
+		'options': { modifiers: 'transform' },
+		'expected': '(?:[\\u0404\\u0454])',
+	},
+	{
+		'pattern': '(?i:[є-ґ])',
+		'options': { modifiers: 'transform' },
+		'matches': ['\u0462', '\u0463', '\u1C87'],
+		'expected': '(?:[\\u0404-\\u040F\\u0454-\\u0491\\u1C87])',
```

Comment:
Here U+1C87 ᲇ should be matched because the uppercase of U+1C87 is U+0462 Ѣ, well in range of `[є-ґ]`, while the lowercase of U+0462 Ѣ is U+0463 ѣ. So the `String#toLowerCase` approach in #80 will miss cases like that, which is why we have introduced `iu-mappings`.